parkingLot.java

```diff
@@ -0,0 +1,167 @@
+import java.io.*;
```

Comment:
Include time and space complexities as well? Also, try implementing this in python.